HpToolsLauncher/Launcher.cs

```diff
@@ -900,6 +917,22 @@ private IAssetRunner CreateRunner(TestStorageType runType, JavaProperties ciPara
                                 }
                             }
 
+                            if (_ciParams.ContainsKey("MobileClientId"))
+                            {
+                                string mcClientId = _ciParams["MobileClientId"];
+                                if (!string.IsNullOrEmpty(mcClientId))
+                                {
+                                    mcConnectionInfo.MobileClientId = mcClientId;
+                                    if (_ciParams.ContainsKey("MobileSecretKey"))
+                                    {
+                                        string mcSecretKey = _ciParams["MobileSecretKey"];
+                                        if (!string.IsNullOrEmpty(mcSecretKey))
+                                        {
+                                            mcConnectionInfo.MobileSecretKey = Decrypt(mcSecretKey, _secretKey);

```

Comment:
The `Decrypt` method only works in Jenkins plugin and doesn't work in the **standalone** tools launcher. You have to create two parameters for this purpose and another one might be `MobileSecretKeyBaiscAuth`. You can refer to the `MobilePassword` parameter and its corresponding _***BasicAuth_ parameter for the code.

As you introduced new parameters, you must also update the README regarding the usage of the new parameters.

Reply:
Updated as suggested. Thank you.